pandas/io/common.py

```diff
@@ -626,6 +626,21 @@ def get_handle(
     ...
 
 
+@overload
+def get_handle(
```

Comment:
Just curious, why do we need this overloads when it's the same as the main function typing?

Reply:
Mypy and pyright ignore annotations of the implementation when overloads are present (probably the expected behavior based on some PEP). The overloads cover "only" the cases when `is_text` `True` or `False` but not the case when it could be either of them.

The reason I fixed it is that pyright expects the ignore comment at a different line than mypy.